storage/storage.go

```diff
@@ -1030,6 +1038,17 @@ func (o *ObjectHandle) ReadCompressed(compressed bool) *ObjectHandle {
 	return &o2
 }
 
+// OverrideUnlockedRetention provides an option for overriding an Unlocked
+// Retention policy. This must be set to true in order to change a policy
+// from Unlocked to Locked, to set it to null, or to reduce its
+// RetainUntil attribute. It is not required for setting the ObjectRetention for
+// the first time nor for extending the RetainUntil time.
+func (o *ObjectHandle) OverrideUnlockedRetention(override bool) *ObjectHandle {
```

Comment:
What does sending false for this query parameter do? (Same question applies for the bucket-level one as well.)

I'm wondering if we should remove the arg from these methods and just set to true if the user calls them. Thoughts?

Reply:
I don't think it does anything now, but I am hesitant to change it. I think we want to /really/ make sure users want to use these query params; in my opinion having the bool passed in adds a small extra step that doesn't really impact user experience negatively very much.

Reply:
Sure, I guess it also mirrors the API itself more closely so there doesn't seem to be much harm either way.